test/e2e/network_test.go

```diff
@@ -260,39 +261,39 @@ func (tc *testContext) testEastWestNetworking(t *testing.T) {
 	}{
 		{
 			name:            "linux curling windows",
-			webserverOS:     windowsOS,
 			curlerOS:        linux,
+			webserverOS:     windowsOS,
```

Comment:
do these need to be moved ?

Reply:
it is not, just moved around to match the struct definition and logic flow, for better readability.